sqlDriver.py

```diff
@@ -0,0 +1,71 @@
+import datetime
+import time
+from enum import Enum
+
+class sqlDriver:
+    def __init__(self, numOfTempSensors, numOfVoltageSensors,
+                 voltageTableName, tempratureTableName, motorTableName,
+                 workConditionTableName,tokens,sql):
+        self.sqlValues =""
+        self.numOfTempSensors =numOfTempSensors
+        self.numOfVoltageSensors = numOfVoltageSensors
+        self.voltageTableName = voltageTableName
+        self.tempratureTableName = tempratureTableName
+        self.motorTableName = motorTableName
+        self.workConditionTableName = workConditionTableName
+        self.voltageArray = [0.0] * numOfVoltageSensors
+        self.tempratureArray = [0.0] * numOfTempSensors
+        self.tokens=tokens
```

Comment:
what is this tokens?

Reply:
Sorry I just realized I uploaded the wrong file for serialReceiver.py. I renamed my version of serialReceiver as test.py and I forgot to put it in the team folder. But I believed token was a string generated in the statement "tokens = line.split(" ")", and I passed the string into my class with constuctor  __init__(self, numOfTempSensors, numOfVoltageSensors,voltageTableName, tempratureTableName, motorTableName,workConditionTableName,tokens,sql). 

I will try to upload test.py as soon as possible.

Reply:
It's there.

Reply:
please directly copy paste content in test.py into serialReceiver.py and do the push. Then we could see the diff between original serialReceiver.py with the modification. And also delete test.py.

Reply:
For `tokens`:
from the design perspective, I do not think you should initialize the class with `tokens` parameter. Instead, pass `tokens` into functions when they are called is a better way.
e.g.
```
if tokens[0] == "voltage":
      table.voltage(tokens)
```